README.md

```diff
@@ -1,86 +1,100 @@
-## FusionAuth Load Tests ![semver 2.0.0 compliant](http://img.shields.io/badge/semver-2.0.0-brightgreen.svg?style=flat-square)
+# FusionAuth Load Tests ![semver 2.0.0 compliant](http://img.shields.io/badge/semver-2.0.0-brightgreen.svg?style=flat-square)
 
-## Examples Usages:
+This repo contains load tests you can run against any FusionAuth instance.
 
-### Build
 
-See the Savant setup below if you do not yet have Savant configured. 
+## Setup
 
-```
-sb int
-```
+### Start FusionAuth
 
-### Setup FusionAuth
+You will need a working FusionAuth instance. If you want to run load tests, you have probably already figured this part out.
 
-If you want to run load tests, you have probably already figured this part out.
+### Configure FusionAuth
 
-### Configure FusionAuth for load tests
- 
-All of this is optional and will depend upon what you want your tests to do. However, to use the provided examples here, start with these setup steps. 
+#### Create an API Key.
 
-1. Setup an API key. 
-   - If you're a FusionAuth developer, run `sb config` which will insert an API key into the database. 
-   - If you're just an everyday average FusionAuth enthusiast, add an API key however you know how. The examples will expect an API key of `bf69486b-4733-4470-a592-f1bfce7af580` but this can be modified to your liking. 
-   - The key must be a super user key or have permissions to add users and applications as well as log users in. It must work for all tenants.
-2. Set up a tenant. The examples will expect a tenant id of `efb21cfc-fa60-46f4-9598-889151e58517` but that can be modified if you want.
-3. Setup an Application
-   - If needed, update `./src/main/script/setupApplication.sh`. Change the name of the host where FusionAuth runs, if not running it locally.
-   - Run `./src/main/script/setupApplication.sh`  
+1. Go to `https://[your.fusionauth.url]/admin/api-key`.
+1. Click the green + button to create a new API Key.
+1. On the **Add API Key** screen, don't modify or input any values, just click the Save icon.
+1. You should now see your API key. Click on the Key name to reveal the key.
 
-You may also want to change the `factor` in the `User-Registrations.json` file. With no changes, it is `1` which is not very realistic. For a typical FusionAuth deployment, the default is `24000`.
+Notes:
+* If you don't want to create a Super User key, make sure the key at least has permissions to add applications, add users, and log in users, for all tenants.
+* If you're a FusionAuth developer, you can run `sb config` to insert an API key into the database.
```

Comment:
This is true. But if you're a FusionAuth Developer, we set up our dev kickstart to have a known tenant, API key, etc that is pre-configured for the load test harness. 

One of the reasons for this is that if you are load testing and have enabled send set up password emails, and you're using a legit SMTP server, you can damage your send reputation very quickly. Ask me how I know. 😎

That is just to say, we could omit this step, since we sort of expect the FusionAuth Dev team to be running this against the dev kickstarted config. 

If you are a normal user outside of FusionAuth dev, we could call out that they should be sure not to use a legit SMTP config unless they are very careful.

Reply:
I can take out the `sb config` step.

The SMTP thing seems like a pretty important potential footgun to mention. I'll add that to the readme.